modules/RTC/TraceablePeerConnection.js

```diff
@@ -1059,7 +1059,7 @@ TraceablePeerConnection.prototype._createRemoteTrack = function(
             + 'deleting the existing track');
         const existingTrack = Array.from(userTracksByMediaType)[0];
 
-        // The existing track needs to be removed here. This happens on Safari sometimes when an SSRC is removed from
+        // The existing track needs to be removed here. This happens on Safari sometimes when a SSRC is removed from
```

Comment:
Just to be pedantic, I think "an SSRC" is correct, because even though it's a consonant, it sounds like a vowel: "an esesarcee"